internal/grpc/services/ocmshareprovider/ocmshareprovider.go

```diff
@@ -299,7 +300,9 @@ func (s *service) CreateOCMShare(ctx context.Context, req *ocm.CreateOCMShareReq
 	newShareReq := &client.NewShareRequest{
 		ShareWith:  formatOCMUser(req.Grantee.GetUserId()),
 		Name:       ocmshare.Name,
-		ResourceID: fmt.Sprintf("%s:%s", req.ResourceId.StorageId, req.ResourceId.OpaqueId),
+		// FIXME: https://github.com/pondersource/nc-sciencemesh/issues/293#issuecomment-1537528109
+		// ResourceID: fmt.Sprintf("%s:%s", req.ResourceId.StorageId, req.ResourceId.OpaqueId),
+		ResourceID: ocmshare.Id.OpaqueId,
```

Comment:
Why this has to be the id of the share from the sender side? Where has this been defined?
This now is named `providerId` in the OCM specs (and we have to update reva accordingly), stating `Identifier to identify the shared resource at the provider side.`